charts/kubernetes-etcd-backup/templates/cronjob.yaml

```
@@ -52,4 +52,11 @@ spec:
             {{- else if .Values.persistence.provisioning.enabled }}
             persistentVolumeClaim:
               claimName: {{ include "kubernetes-etcd-backup.fullname" . }}
+            {{- else if .Values.persistence.existingClaim }}
+            persistentVolumeClaim:
+              claimName: {{ .Values.persistence.existingClaim }}
             {{- end }}
+          {{ if .Values.affinity }}
+            affinity:
+              {{ .Values.affinity | toYaml | nindent 14 }}
+          {{ end }}
```

simple `if` often look nicer as a `with`
```suggestion
          {{ with .Values.affinity }}
            affinity:
              {{ . | toYaml | nindent 14 }}
          {{ end }}
```

---

Cool, i wasnt aware thats a thing